hibernate-core/src/test/java/x/RegexTest.java

```diff
@@ -4,15 +4,18 @@
  */
 package x;
 
+import org.hibernate.dialect.OracleDialect;
 import org.hibernate.testing.orm.junit.EntityManagerFactoryScope;
 import org.hibernate.testing.orm.junit.Jpa;
+import org.hibernate.testing.orm.junit.SkipForDialect;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @Jpa
 class RegexTest {
 	@Test
+	@SkipForDialect( dialectClass = OracleDialect.class, majorVersion = 19)
```

Comment:
Am I missing something? According to [this](https://docs.oracle.com/cd/B12037_01/server.101/b10759/conditions018.htm), that is already supported in Oracle 10.

Reply:
AFAIU, `regexp_like` is a "predicate function" i.e. it can only be used in a context where a condition is required. E.g. `select case when regexp_like(...) then true else false end from dual` might work.